apps/parsley/src/components/Badge/TestStatusBadge/__snapshots__/TestStatusBadge_AllBadges.storyshot

```diff
@@ -30,16 +27,4 @@ exports[`Snapshot Tests TestStatusBadge.stories AllBadges 1`] = `
       Pass
     </div>
   </div>
-</div>
-`;
-
-exports[`Snapshot Tests TestStatusBadge.stories Default 1`] = `
-<div>
-  <div
-    class="leafygreen-ui-n4itms"
-    data-cy="test-status-badge"
-  >
-    Pass
-  </div>
-</div>
-`;
+</div>
```

Comment:
Is it possible to make all of the snapshots end with new lines?

Reply:
I think maybe if we implemented a custom snapshot serializer, but that doesn't seem worth it to me 😅